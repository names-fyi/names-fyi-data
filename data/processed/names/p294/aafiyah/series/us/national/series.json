[{"y":2016,"t":5,"m":0,"f":5,"mr":null,"fr":16200,"mp":0,"fp":0.0000028247215107062596},{"y":2017,"t":5,"m":0,"f":5,"mr":null,"fr":15885,"mp":0,"fp":0.00000289979133101582},{"y":2018,"t":6,"m":0,"f":6,"mr":null,"fr":13774,"mp":0,"fp":0.0000035327928552797296},{"y":2019,"t":6,"m":0,"f":6,"mr":null,"fr":13775,"mp":0,"fp":0.0000035817923169360873},{"y":2020,"t":12,"m":0,"f":12,"mr":null,"fr":8385,"mp":0,"fp":0.000007442354314363806},{"y":2021,"t":10,"m":0,"f":10,"mr":null,"fr":9536,"mp":0,"fp":0.0000061132167746668295},{"y":2022,"t":9,"m":0,"f":9,"mr":null,"fr":10333,"mp":0,"fp":0.0000055257777532187655}]