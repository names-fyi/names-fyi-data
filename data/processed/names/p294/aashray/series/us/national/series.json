[{"y":2013,"t":5,"m":5,"f":0,"mr":12134,"fr":null,"mp":0.0000026451681506941714,"fp":0},{"y":2015,"t":9,"m":9,"f":0,"mr":8036,"fr":null,"mp":0.0000046997413575672885,"fp":0}]