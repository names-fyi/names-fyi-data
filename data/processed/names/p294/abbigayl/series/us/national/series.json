[{"y":1999,"t":6,"m":0,"f":6,"mr":null,"fr":12693,"mp":0,"fp":0.0000033828354927099894},{"y":2005,"t":6,"m":0,"f":6,"mr":null,"fr":14510,"mp":0,"fp":0.0000032478428910132187},{"y":2006,"t":5,"m":0,"f":5,"mr":null,"fr":17183,"mp":0,"fp":0.000002630647144458831},{"y":2007,"t":7,"m":0,"f":7,"mr":null,"fr":13936,"mp":0,"fp":0.00000364215134595703},{"y":2008,"t":5,"m":0,"f":5,"mr":null,"fr":17518,"mp":0,"fp":0.0000026453472838367694}]