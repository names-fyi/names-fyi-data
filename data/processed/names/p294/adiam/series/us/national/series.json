[{"y":1993,"t":5,"m":0,"f":5,"mr":null,"fr":13454,"mp":0,"fp":0.0000027640786966373878},{"y":1996,"t":8,"m":0,"f":8,"mr":null,"fr":9589,"mp":0,"fp":0.000004563993174548207},{"y":2013,"t":5,"m":0,"f":5,"mr":null,"fr":16586,"mp":0,"fp":0.0000028515814585611032}]