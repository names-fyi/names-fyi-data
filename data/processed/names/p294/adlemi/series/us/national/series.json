[{"y":2001,"t":5,"m":0,"f":5,"mr":null,"fr":15389,"mp":0,"fp":0.0000027771714286825154},{"y":2006,"t":5,"m":0,"f":5,"mr":null,"fr":17218,"mp":0,"fp":0.000002630647144458831},{"y":2007,"t":6,"m":0,"f":6,"mr":null,"fr":15532,"mp":0,"fp":0.0000031218440108203113},{"y":2014,"t":5,"m":0,"f":5,"mr":null,"fr":16554,"mp":0,"fp":0.000002801263257678683}]