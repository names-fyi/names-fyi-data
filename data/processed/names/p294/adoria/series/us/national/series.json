[{"y":1935,"t":5,"m":0,"f":5,"mr":null,"fr":4209,"mp":0,"fp":0.0000047685080101397556},{"y":1956,"t":5,"m":0,"f":5,"mr":null,"fr":5954,"mp":0,"fp":0.000002490336250181172},{"y":1962,"t":5,"m":0,"f":5,"mr":null,"fr":6581,"mp":0,"fp":0.000002542521123265492},{"y":1988,"t":6,"m":0,"f":6,"mr":null,"fr":10390,"mp":0,"fp":0.0000033694349569891626},{"y":1992,"t":12,"m":0,"f":12,"mr":null,"fr":7061,"mp":0,"fp":0.000006508202776073894},{"y":1993,"t":7,"m":0,"f":7,"mr":null,"fr":10606,"mp":0,"fp":0.000003869710175292343},{"y":1994,"t":6,"m":0,"f":6,"mr":null,"fr":11790,"mp":0,"fp":0.0000033602526910023635},{"y":1996,"t":6,"m":0,"f":6,"mr":null,"fr":11823,"mp":0,"fp":0.0000034229948809111556},{"y":1998,"t":8,"m":0,"f":8,"mr":null,"fr":10138,"mp":0,"fp":0.000004527988344958},{"y":1999,"t":5,"m":0,"f":5,"mr":null,"fr":14435,"mp":0,"fp":0.0000028190295772583248},{"y":2000,"t":7,"m":0,"f":7,"mr":null,"fr":11823,"mp":0,"fp":0.000003854988547930449},{"y":2007,"t":7,"m":0,"f":7,"mr":null,"fr":13958,"mp":0,"fp":0.00000364215134595703},{"y":2008,"t":6,"m":0,"f":6,"mr":null,"fr":15441,"mp":0,"fp":0.000003174416740604123},{"y":2013,"t":7,"m":0,"f":7,"mr":null,"fr":13173,"mp":0,"fp":0.000003992214041985545},{"y":2017,"t":5,"m":0,"f":5,"mr":null,"fr":15945,"mp":0,"fp":0.00000289979133101582},{"y":2021,"t":7,"m":0,"f":7,"mr":null,"fr":12204,"mp":0,"fp":0.000004279251742266781},{"y":2022,"t":5,"m":0,"f":5,"mr":null,"fr":15258,"mp":0,"fp":0.000003069876529565981}]