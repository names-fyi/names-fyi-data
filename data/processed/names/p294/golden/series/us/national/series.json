[{"y":1880,"t":7,"m":7,"f":0,"mr":772,"fr":null,"mp":0.00006335414969680514,"fp":0},{"y":1881,"t":5,"m":5,"f":0,"mr":924,"fr":null,"mp":0.00004963419597565939,"fp":0},{"y":1883,"t":5,"m":5,"f":0,"mr":953,"fr":null,"mp":0.00004778972520908005,"fp":0},{"y":1884,"t":13,"m":5,"f":8,"mr":1038,"fr":816,"mp":0.00004369025357823177,"fp":0.00006200637115463613},{"y":1885,"t":9,"m":0,"f":9,"mr":null,"fr":766,"mp":0,"fp":0.00006764120100710233},{"y":1886,"t":21,"m":8,"f":13,"mr":720,"fr":645,"mp":0.00007221259387637204,"fp":0.00008994485688389502},{"y":1887,"t":15,"m":7,"f":8,"mr":781,"fr":889,"mp":0.00006902604253976393,"fp":0.00005480127686975106},{"y":1888,"t":19,"m":10,"f":9,"mr":668,"fr":916,"mp":0.00008274652257738868,"fp":0.000050385730761048474},{"y":1889,"t":20,"m":9,"f":11,"mr":693,"fr":823,"mp":0.00008138903960933261,"fp":0.00006167094625657356},{"y":1890,"t":27,"m":9,"f":18,"mr":694,"fr":624,"mp":0.00008106355382619974,"fp":0.00009454923651491515},{"y":1891,"t":22,"m":6,"f":16,"mr":893,"fr":680,"mp":0.0000592938107143916,"fp":0.00008626266982963122},{"y":1892,"t":21,"m":0,"f":21,"mr":null,"fr":627,"mp":0,"fp":0.00009889845953875643},{"y":1893,"t":25,"m":10,"f":15,"mr":673,"fr":774,"mp":0.00008903292437543404,"fp":0.00007045395833822597},{"y":1894,"t":20,"m":11,"f":9,"mr":648,"fr":1097,"mp":0.00009501762144979614,"fp":0.00004037286584545267},{"y":1895,"t":27,"m":11,"f":16,"mr":652,"fr":799,"mp":0.00009370155203843468,"fp":0.00006848523501136427},{"y":1896,"t":25,"m":8,"f":17,"mr":852,"fr":799,"mp":0.00006690865296154425,"fp":0.00007145318975445322},{"y":1897,"t":28,"m":7,"f":21,"mr":885,"fr":684,"mp":0.0000620804029905017,"fp":0.000089666951323655},{"y":1898,"t":42,"m":18,"f":24,"mr":493,"fr":671,"mp":0.00014671600671633275,"fp":0.00009274645438033775},{"y":1899,"t":15,"m":7,"f":8,"mr":894,"fr":1220,"mp":0.0000659071650503719,"fp":0.00003433137501448355},{"y":1900,"t":39,"m":13,"f":26,"mr":709,"fr":711,"mp":0.00008639251442090433,"fp":0.00008672882232266458},{"y":1901,"t":21,"m":9,"f":12,"mr":750,"fr":974,"mp":0.00008453323565048324,"fp":0.000050136831770039064},{"y":1902,"t":33,"m":14,"f":19,"mr":599,"fr":804,"mp":0.00011413756838063249,"fp":0.000071949801759362},{"y":1903,"t":32,"m":6,"f":26,"mr":1065,"fr":659,"mp":0.00005032248324680662,"fp":0.00009924762664569743},{"y":1904,"t":35,"m":17,"f":18,"mr":546,"fr":862,"mp":0.00013268603362420194,"fp":0.00006536825935220055},{"y":1905,"t":38,"m":15,"f":23,"mr":622,"fr":765,"mp":0.00011337268625242806,"fp":0.00007887030293054612},{"y":1906,"t":44,"m":16,"f":28,"mr":600,"fr":695,"mp":0.00012016161737536988,"fp":0.00009481979159964375},{"y":1907,"t":45,"m":18,"f":27,"mr":603,"fr":731,"mp":0.0001225940909648155,"fp":0.00008475480748102434},{"y":1908,"t":35,"m":14,"f":21,"mr":724,"fr":860,"mp":0.00009070941239738497,"fp":0.00006281538558177517},{"y":1909,"t":51,"m":26,"f":25,"mr":510,"fr":820,"mp":0.00015854045220615136,"fp":0.00007199880193993572},{"y":1910,"t":46,"m":21,"f":25,"mr":643,"fr":864,"mp":0.0001081309311102987,"fp":0.0000630512253375132},{"y":1911,"t":45,"m":17,"f":28,"mr":805,"fr":818,"mp":0.00007523122538390052,"fp":0.00006693616441434441},{"y":1912,"t":72,"m":40,"f":32,"mr":675,"fr":896,"mp":0.00009303602122151644,"fp":0.000057335442799833014},{"y":1913,"t":63,"m":39,"f":24,"mr":751,"fr":1131,"mp":0.00007608791080155687,"fp":0.0000384279140367563},{"y":1914,"t":75,"m":40,"f":35,"mr":860,"fr":1010,"mp":0.00006109071360062557,"fp":0.00004595696852078962},{"y":1915,"t":92,"m":45,"f":47,"mr":931,"fr":974,"mp":0.00005302995485383177,"fp":0.000047771364218028506},{"y":1916,"t":104,"m":48,"f":56,"mr":920,"fr":908,"mp":0.00005392749448366671,"fp":0.00005362269159100441},{"y":1917,"t":90,"m":53,"f":37,"mr":879,"fr":1194,"mp":0.00005726714691832516,"fp":0.00003421908352045931},{"y":1918,"t":98,"m":40,"f":58,"mr":1085,"fr":940,"mp":0.00003946789385504627,"fp":0.000050102450873683084},{"y":1919,"t":128,"m":66,"f":62,"mr":805,"fr":895,"mp":0.00006733807488606092,"fp":0.00005486133355513829},{"y":1920,"t":95,"m":51,"f":44,"mr":967,"fr":1143,"mp":0.00004791620864408404,"fp":0.00003671892994361975},{"y":1921,"t":116,"m":58,"f":58,"mr":925,"fr":977,"mp":0.000052657860093513095,"fp":0.000047037680425999185},{"y":1922,"t":94,"m":44,"f":50,"mr":1063,"fr":1082,"mp":0.00004043534159133287,"fp":0.00004163214667837918},{"y":1923,"t":80,"m":48,"f":32,"mr":1016,"fr":1439,"mp":0.00004379086577278562,"fp":0.000026525748627085275},{"y":1924,"t":98,"m":54,"f":44,"mr":954,"fr":1193,"mp":0.00004767146942007657,"fp":0.000035231906014086355},{"y":1925,"t":75,"m":44,"f":31,"mr":1049,"fr":1481,"mp":0.00003942892577683945,"fp":0.00002546433391463205},{"y":1926,"t":60,"m":36,"f":24,"mr":1202,"fr":1745,"mp":0.00003241785899852229,"fp":0.000020246381591989858},{"y":1927,"t":67,"m":44,"f":23,"mr":1068,"fr":1787,"mp":0.0000390440940927173,"fp":0.000019288441024333626},{"y":1928,"t":71,"m":38,"f":33,"mr":1152,"fr":1349,"mp":0.000034308383614677124,"fp":0.000028613543744038844},{"y":1929,"t":58,"m":34,"f":24,"mr":1205,"fr":1611,"mp":0.00003160967627972704,"fp":0.000021494802288121705},{"y":1930,"t":57,"m":38,"f":19,"mr":1141,"fr":1898,"mp":0.000034628342204567845,"fp":0.00001687519373166488},{"y":1931,"t":50,"m":28,"f":22,"mr":1316,"fr":1650,"mp":0.000026932684640386138,"fp":0.000020662728851462216},{"y":1932,"t":44,"m":29,"f":15,"mr":1275,"fr":2175,"mp":0.00002777415968800999,"fp":0.000014056433770301005},{"y":1933,"t":55,"m":34,"f":21,"mr":1133,"fr":1675,"mp":0.00003429192717604971,"fp":0.000020834966232472583},{"y":1934,"t":34,"m":22,"f":12,"mr":1476,"fr":2484,"mp":0.00002130624728224289,"fp":0.000011491996303407856},{"y":1935,"t":37,"m":19,"f":18,"mr":1616,"fr":1872,"mp":0.000018246895866886013,"fp":0.00001716662883650312},{"y":1936,"t":38,"m":26,"f":12,"mr":1305,"fr":2428,"mp":0.000025068093619688094,"fp":0.000011534579708751863},{"y":1937,"t":49,"m":34,"f":15,"mr":1131,"fr":2124,"mp":0.00003187669344933949,"fp":0.000014097558866706641},{"y":1938,"t":38,"m":24,"f":14,"mr":1383,"fr":2272,"mp":0.00002163864011966168,"fp":0.000012687641150007794},{"y":1939,"t":38,"m":30,"f":8,"mr":1206,"fr":3250,"mp":0.000027105070093711263,"fp":0.000007295269655964202},{"y":1940,"t":30,"m":15,"f":15,"mr":1837,"fr":2144,"mp":0.000012940918667188905,"fp":0.000013117793412244148},{"y":1941,"t":31,"m":15,"f":16,"mr":1848,"fr":2130,"mp":0.000012212567057170282,"fp":0.00001324396964813256},{"y":1942,"t":29,"m":21,"f":8,"mr":1555,"fr":3546,"mp":0.000015205810647108615,"fp":0.000005922964443704074},{"y":1943,"t":21,"m":16,"f":5,"mr":1865,"fr":4938,"mp":0.000011210361737353836,"fp":0.0000035835743287069387},{"y":1944,"t":36,"m":21,"f":15,"mr":1498,"fr":2328,"mp":0.000015409315444830248,"fp":0.000011302031728570407},{"y":1945,"t":24,"m":17,"f":7,"mr":1652,"fr":3794,"mp":0.000012632135856391936,"fp":0.000005355469272995046},{"y":1946,"t":34,"m":20,"f":14,"mr":1617,"fr":2655,"mp":0.000012315536357002769,"fp":0.000008909916291336443},{"y":1947,"t":36,"m":24,"f":12,"mr":1533,"fr":3129,"mp":0.00001311843873087851,"fp":0.000006769197443951045},{"y":1948,"t":29,"m":21,"f":8,"mr":1629,"fr":4000,"mp":0.000011968560301881287,"fp":0.000004711749867334793},{"y":1949,"t":29,"m":19,"f":10,"mr":1726,"fr":3474,"mp":0.000010711219325745656,"fp":0.0000058449726221482375},{"y":1950,"t":22,"m":15,"f":7,"mr":1967,"fr":4470,"mp":0.000008378236023845577,"fp":0.000004086276473963706},{"y":1951,"t":25,"m":20,"f":5,"mr":1694,"fr":5714,"mp":0.000010622008908678871,"fp":0.0000027764173332844683},{"y":1952,"t":45,"m":32,"f":13,"mr":1329,"fr":3090,"mp":0.000016463471415554997,"fp":0.000007010591925847351},{"y":1953,"t":20,"m":12,"f":8,"mr":2332,"fr":4367,"mp":0.000006084223911608395,"fp":0.000004252389976053729},{"y":1954,"t":26,"m":19,"f":7,"mr":1787,"fr":4852,"mp":0.000009319317080444346,"fp":0.000003604653917177412},{"y":1955,"t":17,"m":12,"f":5,"mr":2355,"fr":6173,"mp":0.000005825717825531403,"fp":0.0000025572360574375676},{"y":1956,"t":26,"m":16,"f":10,"mr":2038,"fr":4007,"mp":0.000007563853102408898,"fp":0.000004980672500362344},{"y":1957,"t":24,"m":17,"f":7,"mr":1982,"fr":5076,"mp":0.000007879292940060828,"fp":0.0000034230833178479565},{"y":1958,"t":27,"m":22,"f":5,"mr":1700,"fr":6433,"mp":0.000010370656698265744,"fp":0.0000024858344722598275},{"y":1959,"t":9,"m":9,"f":0,"mr":2905,"fr":null,"mp":0.00000421815154974888,"fp":0},{"y":1960,"t":16,"m":11,"f":5,"mr":2597,"fr":6700,"mp":0.000005158918127969309,"fp":0.0000024727246111764184},{"y":1961,"t":9,"m":0,"f":9,"mr":null,"fr":4660,"mp":0,"fp":0.000004459976173816173},{"y":1962,"t":18,"m":12,"f":6,"mr":2429,"fr":6115,"mp":0.000005801316512093811,"fp":0.0000030510253479185905},{"y":1963,"t":28,"m":21,"f":7,"mr":1761,"fr":5513,"mp":0.000010338828010149776,"fp":0.0000036318130218137067},{"y":1964,"t":18,"m":11,"f":7,"mr":2589,"fr":5655,"mp":0.000005519766283059637,"fp":0.0000036942530616122246},{"y":1965,"t":12,"m":7,"f":5,"mr":3362,"fr":6867,"mp":0.000003762427432180902,"fp":0.0000028326621812405247},{"y":1966,"t":9,"m":9,"f":0,"mr":2864,"fr":null,"mp":0.000005046090429304554,"fp":0},{"y":1967,"t":14,"m":6,"f":8,"mr":3698,"fr":5146,"mp":0.000003440315683367106,"fp":0.0000048455951117636514},{"y":1968,"t":14,"m":7,"f":7,"mr":3482,"fr":5838,"mp":0.000004027036371617217,"fp":0.000004267496430544057},{"y":1969,"t":6,"m":6,"f":0,"mr":4122,"fr":null,"mp":0.0000033544983543390156,"fp":0},{"y":1970,"t":18,"m":11,"f":7,"mr":2966,"fr":6572,"mp":0.000005918146651674028,"fp":0.0000040042582426225835},{"y":1971,"t":8,"m":8,"f":0,"mr":3764,"fr":null,"mp":0.000004523078158225189,"fp":0},{"y":1972,"t":9,"m":9,"f":0,"mr":3439,"fr":null,"mp":0.00000554836323284631,"fp":0},{"y":1973,"t":10,"m":10,"f":0,"mr":3293,"fr":null,"mp":0.000006414767307523303,"fp":0},{"y":1974,"t":11,"m":11,"f":0,"mr":3207,"fr":null,"mp":0.000006994429890378208,"fp":0},{"y":1975,"t":13,"m":5,"f":8,"mr":5778,"fr":6717,"mp":0.000003201413744309487,"fp":0.000005487269534679544},{"y":1976,"t":23,"m":23,"f":0,"mr":2078,"fr":null,"mp":0.000014654665981932708,"fp":0},{"y":1977,"t":32,"m":23,"f":9,"mr":2090,"fr":6521,"mp":0.000013994311008350953,"fp":0.0000058701598901106065},{"y":1978,"t":29,"m":23,"f":6,"mr":2097,"fr":8906,"mp":0.00001400492851701812,"fp":0.000003916674064828789},{"y":1979,"t":38,"m":27,"f":11,"mr":1964,"fr":5872,"mp":0.000015679906896197276,"fp":0.000006852028574204978},{"y":1980,"t":26,"m":20,"f":6,"mr":2472,"fr":9482,"mp":0.000011208217865338866,"fp":0.0000036135130935646946},{"y":1981,"t":26,"m":15,"f":11,"mr":2951,"fr":6018,"mp":0.000008372619524725461,"fp":0.000006594589798529286},{"y":1982,"t":15,"m":9,"f":6,"mr":4354,"fr":9608,"mp":0.000004959087527894867,"fp":0.0000035433140616418537},{"y":1983,"t":25,"m":13,"f":12,"mr":3298,"fr":5569,"mp":0.000007251105793633529,"fp":0.0000071830050101459944},{"y":1984,"t":21,"m":13,"f":8,"mr":3342,"fr":7624,"mp":0.000007202232027106985,"fp":0.00000475132740209296},{"y":1985,"t":23,"m":14,"f":9,"mr":3275,"fr":7150,"mp":0.00000757473285540376,"fp":0.000005230234942153601},{"y":1986,"t":16,"m":10,"f":6,"mr":4242,"fr":9956,"mp":0.000005430435309124815,"fp":0.0000034975674418441972},{"y":1987,"t":19,"m":10,"f":9,"mr":4364,"fr":7575,"mp":0.000005356487643922127,"fp":0.0000051767455986158535},{"y":1988,"t":18,"m":9,"f":9,"mr":4938,"fr":7929,"mp":0.00000470208380680705,"fp":0.000005054152435483744},{"y":1989,"t":22,"m":13,"f":9,"mr":4114,"fr":8270,"mp":0.000006497378307852782,"fp":0.0000048820284936880795},{"y":1990,"t":10,"m":10,"f":0,"mr":5132,"fr":null,"mp":0.000004869257988382924,"fp":0},{"y":1991,"t":22,"m":16,"f":6,"mr":3711,"fr":12026,"mp":0.000007920407821798745,"fp":0.0000032001877443476684},{"y":1992,"t":16,"m":8,"f":8,"mr":6336,"fr":9796,"mp":0.0000040057342085194955,"fp":0.00000433880185071593},{"y":1993,"t":8,"m":8,"f":0,"mr":6421,"fr":null,"mp":0.000004079422272217808,"fp":0},{"y":1994,"t":5,"m":5,"f":0,"mr":9318,"fr":null,"mp":0.0000025882918102305857,"fp":0},{"y":1995,"t":10,"m":10,"f":0,"mr":5646,"fr":null,"mp":0.000005253599109409879,"fp":0},{"y":1996,"t":17,"m":10,"f":7,"mr":5688,"fr":10978,"mp":0.000005279644278687079,"fp":0.000003993494027729681},{"y":1997,"t":10,"m":10,"f":0,"mr":5793,"fr":null,"mp":0.000005305127033919391,"fp":0},{"y":1998,"t":16,"m":10,"f":6,"mr":5985,"fr":13003,"mp":0.000005232336677842938,"fp":0.0000033959912587185},{"y":1999,"t":8,"m":8,"f":0,"mr":7391,"fr":null,"mp":0.000004166759984728825,"fp":0},{"y":2000,"t":16,"m":16,"f":0,"mr":4655,"fr":null,"mp":0.000008147514829750039,"fp":0},{"y":2001,"t":16,"m":11,"f":5,"mr":6167,"fr":16198,"mp":0.000005662219765162009,"fp":0.0000027771714286825154},{"y":2002,"t":20,"m":10,"f":10,"mr":6708,"fr":9750,"mp":0.000005151434133505597,"fp":0.000005566109517657648},{"y":2003,"t":8,"m":8,"f":0,"mr":8105,"fr":null,"mp":0.000004050694440928217,"fp":0},{"y":2004,"t":23,"m":17,"f":6,"mr":4858,"fr":14785,"mp":0.000008566389518770471,"fp":0.000003268224847333047},{"y":2005,"t":23,"m":8,"f":15,"mr":8550,"fr":7638,"mp":0.000004006436339980178,"fp":0.000008119607227533047},{"y":2006,"t":28,"m":12,"f":16,"mr":6602,"fr":7611,"mp":0.000005840427830806699,"fp":0.000008418070862268259},{"y":2007,"t":32,"m":17,"f":15,"mr":5277,"fr":8231,"mp":0.000008193928106474794,"fp":0.000007804610027050779},{"y":2008,"t":19,"m":11,"f":8,"mr":7368,"fr":13004,"mp":0.0000053939627826375204,"fp":0.000004232555654138831},{"y":2009,"t":18,"m":11,"f":7,"mr":7374,"fr":14194,"mp":0.000005548244762330848,"fp":0.00000381219182513585},{"y":2010,"t":27,"m":18,"f":9,"mr":5107,"fr":11588,"mp":0.000009385793662712118,"fp":0.000005065753480172641},{"y":2011,"t":29,"m":21,"f":8,"mr":4555,"fr":12566,"mp":0.000011063118250510089,"fp":0.0000045503409911780264},{"y":2012,"t":33,"m":23,"f":10,"mr":4297,"fr":10694,"mp":0.000012131979052819471,"fp":0.000005682150716774902},{"y":2013,"t":33,"m":20,"f":13,"mr":4675,"fr":8735,"mp":0.000010580672602776686,"fp":0.000007414111792258869},{"y":2014,"t":40,"m":24,"f":16,"mr":4174,"fr":7424,"mp":0.000012504708804409161,"fp":0.000008964042424571784},{"y":2015,"t":45,"m":28,"f":17,"mr":3810,"fr":7079,"mp":0.000014621417556876009,"fp":0.000009532412726556012},{"y":2016,"t":36,"m":17,"f":19,"mr":5385,"fr":6505,"mp":0.000008967313614386524,"fp":0.000010733941740683786},{"y":2017,"t":66,"m":32,"f":34,"mr":3468,"fr":4240,"mp":0.000017311370974024287,"fp":0.000019718581050907577},{"y":2018,"t":66,"m":31,"f":35,"mr":3554,"fr":4097,"mp":0.000017095176568358372,"fp":0.00002060795832246509},{"y":2019,"t":81,"m":44,"f":37,"mr":2816,"fr":3945,"mp":0.0000245716436530886,"fp":0.000022087719287772536},{"y":2020,"t":70,"m":38,"f":32,"mr":3034,"fr":4286,"mp":0.000022072644558396086,"fp":0.000019846278171636817},{"y":2021,"t":83,"m":48,"f":35,"mr":2660,"fr":4115,"mp":0.00002752430883880102,"fp":0.000021396258711333904},{"y":2022,"t":87,"m":45,"f":42,"mr":2824,"fr":3646,"mp":0.000025964044990497158,"fp":0.00002578696284835424}]